UnitTests/GitCommands.Tests/Git/GetAllChangedFilesOutputParserTest.cs

```diff
@@ -1,5 +1,7 @@
-﻿using GitCommands;
+﻿using System.Windows.Forms;
+using GitCommands;
 using GitCommands.Git;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
 using Newtonsoft.Json;
```

Comment:
🤔 Are all of these in use?

Reply:
System.Windows.Forms was not, the others were